apps/web/src/app/(default)/auth/user/[id]/layout.tsx

```diff
@@ -4,11 +4,12 @@ import {
   SidebarLayoutContent,
   SidebarLayoutRoot,
 } from "@/components/sidebar-layout";
+import { getUser } from "@/lib/get-user";
 
 const routes = [
```

Comment:
Jeg hadde bare gjort dette til en funksjon som tar inn bruker-iden som argument.

```suggestion
const createRoutes = (userId: string) => [
```